### ISSUE TITLE
M365 Pro: Change currents for Drive mode not active

### ISSUE THREAD

**Issue by vobele** [BotoX/xiaomi-m365-firmware-patcher#75]
Why for M365 Pro, current settings for Drive mode can't be patched? :(

It would be great to patch max speed AND phase/battery currents in Drive mode for a balanced drive (and patch all settings in Sport mode for a sportive drive). Thanks

**Comment by flpxra** [comment]
Yes current settings for drive mode does not work even if you generate firmware by entering the url directly: [http://91.223.103.22/cfw?version=DRV155&current_drive_params=on&current_drive_phase=55000&current_drive_battery=25000](http://91.223.103.22/cfw?version=DRV155&current_drive_params=on&current_drive_phase=55000&current_drive_battery=25000)
Are there any plans for this to work?
